tests/integ_tests/simple_wf_tests.rs

```diff
@@ -334,6 +335,177 @@ fn activity_completion_req(
         task.task_token,
     )
 }
+fn schedule_activity_and_timer_cmds(
+    task_q: &str,
+    activity_id: &str,
+    timer_id: &str,
+    cancellation_type: ActivityCancellationType,
+    task: WfActivation,
+) -> WfActivationCompletion {
+    WfActivationCompletion::ok_from_cmds(
+        vec![
+            ScheduleActivity {
+                activity_id: activity_id.to_string(),
+                activity_type: "test_activity".to_string(),
+                namespace: NAMESPACE.to_owned(),
+                task_queue: task_q.to_owned(),
+                schedule_to_start_timeout: Some(Duration::from_secs(30).into()),
+                start_to_close_timeout: Some(Duration::from_secs(30).into()),
+                schedule_to_close_timeout: Some(Duration::from_secs(60).into()),
+                heartbeat_timeout: Some(Duration::from_secs(60).into()),
+                cancellation_type: cancellation_type as i32,
+                ..Default::default()
+            }
+            .into(),
+            StartTimer {
+                timer_id: timer_id.to_string(),
+                start_to_fire_timeout: Some(Duration::from_millis(50).into()),
+            }
+            .into(),
+        ],
+        task.task_token,
+    )
+}
+
+#[tokio::test]
+async fn activity_cancellation() {
```

Comment:
This one should probably be named `activity_cancellation_try_cancel`

Reply:
Yes let's do that, I named it without the suffix initially cause that's a default value, but making it more explicit makes sense.

Reply:
Done